DiamondLightSource/python-artemis
# Add PandA grid scan and setup plans

Fixes #https://github.com/DiamondLightSource/hyperion/issues/870

After this is merged, there are some follow-up issues to add a cleaner way to switch between regular fast grid scans and panda grid scans. The dodal change for this PR pins ophyd-async to a branch which works with the panda and with hyperion

Link to dodal PR (if required): #[279](https://github.com/DiamondLightSource/dodal/pull/279)
(remember to update `setup.cfg` with the dodal commit tag if you need it for tests to pass!)

### To test:
1. Check that use_panda property makes sense and that Hyperion runs as it would do ordinarily with this set to false.
2. Check new unit tests
